fast64_internal/oot/data/oot_getters.py

```diff
@@ -0,0 +1,38 @@
+from xml.etree.ElementTree import parse as XMLParse, Element
```

Comment:
```suggestion
from xml.etree.ElementTree import parse as parseXML, Element
```
I thought XMLParse was a fancy class :sweat_smile: